strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/providers/layout/MavenFileSystem.java

```diff
@@ -39,17 +40,28 @@
     @RepositoryIndexCreatorQualifier(RepositoryTypeEnum.GROUP)
     private RepositoryIndexCreator groupRepositoryIndexCreator;
 
+    private final Repository repository;
+
     public MavenFileSystem(PropertiesBooter propertiesBooter,
                            Repository repository,
                            FileSystem storageFileSystem,
                            LayoutFileSystemProvider provider)
     {
         super(propertiesBooter, repository, storageFileSystem, provider);
+        this.repository = repository;
     }
 
     @Override
     public Set<String> getDigestAlgorithmSet()
     {
+        MavenRepositoryConfiguration mavenRepositoryConfiguration = (MavenRepositoryConfiguration) repository
+                .getRepositoryConfiguration();
+        if (mavenRepositoryConfiguration != null) {
+            Set<String> repoDigestAlgorithms = mavenRepositoryConfiguration.getDigestAlgorithmSet();
+            if (repoDigestAlgorithms != null && repoDigestAlgorithms.size() > 0) {
+                return repoDigestAlgorithms;
+            }
+        }
```

Comment:
Cosmetic: Code style.
```suggestion
        MavenRepositoryConfiguration mavenRepositoryConfiguration = (MavenRepositoryConfiguration) repository.getRepositoryConfiguration();
        if (mavenRepositoryConfiguration != null)
        {
            Set<String> repoDigestAlgorithms = mavenRepositoryConfiguration.getDigestAlgorithmSet();
            if (repoDigestAlgorithms != null && repoDigestAlgorithms.size() > 0)
            {
                return repoDigestAlgorithms;
            }
        }
```